### PR TITLE
Update-delete PR

### PR DESCRIPTION
**Pull request by gilberthappi**
> # Project requirements
> * Add a column `species` of type `string` to your `animals` table. Modify your `schema.sql` file.
> * Insert the following data:
>   
>   * Animal: His name is Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg. He is not neutered and he has never tried to escape.
>   * Animal: Her name is Plantmon. She was born on Nov 15th, 2021, and currently weighs -5.7kg. She is neutered and she has tried to escape 2 times.
>   * Animal: His name is Squirtle. He was born on Apr 2nd, 1993, and currently weighs -12.13kg. He was not neutered and he has tried to escape 3 times.
>   * Animal: His name is Angemon. He was born on Jun 12th, 2005, and currently weighs -45kg. He is neutered and he has tried to escape once.
>   * Animal: His name is Boarmon. He was born on Jun 7th, 2005, and currently weighs 20.4kg. He is neutered and he has tried to escape 7 times.
>   * Animal: Her name is Blossom. She was born on Oct 13th, 1998, and currently weighs 17kg. She is neutered and she has tried to escape 3 times.
>   * Animal: His name is Ditto. He was born on May 14th, 2022, and currently weighs 22kg. He is neutered and he has tried to escape 4 times.
> 
> * Inside a transaction update the `animals` table by setting the `species` column to `unspecified`. Verify that change was made. Then roll back the change and verify that the `species` columns went back to the state before the transaction.
>   ROLLBACK
> * Inside a transaction:
>   
>   * Update the `animals` table by setting the `species` column to `digimon` for all animals that have a name ending in `mon`.
>   * Update the `animals` table by setting the `species` column to `pokemon` for all animals that don't have `species` already set.
>   * Commit the transaction.
>     COMMIT
> * Inside a transaction, I delete all records in the `animals` table, and then I roll back the transaction.
> * After the rollback, I verify if all records in the `animals` table still exists.
>   DELETE-ROLLBACK
> * Inside a transaction:
>   
>   * Delete all animals born after Jan 1st, 2022.
>   * Create a savepoint for the transaction.
>   * Update all animals' weight to be their weight multiplied by -1.
>   * Rollback to the savepoint
>   * Update all animals' weights that are negative to be their weight multiplied by -1.
>   * Commit transaction
>     SAVEPOINTS
> 
> * Write queries to answer the following questions:
>   
>   * How many animals are there?
> 
> TOTAL_ANIMALS
> 
> * How many animals have never tried to escape?
> 
> NEVER_TRIED_TO_ESCAPE
> 
> * What is the average weight of animals?
> 
> AVERAGE_WEIGHT
> 
> * Who escapes the most, neutered or not neutered animals?
> 
> MOST_ESCAPE
> 
> * What is the minimum and maximum weight of each type of animal?
> 
> MAX_MIN_WEIGHT
> 
> * What is the average number of escape attempts per animal type of those born between 1990 and 2000?
> 
> AVERAGE_ESCAPE_ATTENPTS

### CODE REVIEW

File: queries.sql
Comment (Efiamotu-1) [inline queries.sql:1]: - [ ] The project requirement shown in the image below requires you to add your screenshots of your actions, please kindly add this to your pull request :+1: :sparkles: 
erf
- Below is an example of me testing your actions with screenshot of each steps, please kindly use this as a guide :+1: 
cw

**Review by Efiamotu-1 — CHANGES_REQUESTED** [review]
Hi @gilberthappi , 

 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- ✔️ Descriptive PR title and description
- The code is top-notch! 💯 and has a huge potential.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@Efiamotu-1](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [@EFYAMOTU](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[Habeeb Efiamotu Musa](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [Habeeb Efiamotu Musa](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql
Comment (gilberthappi) [reply queries.sql:1]: Okey Thanks

File: README.md [hunk 1]
```diff
@@ -1,23 +1,208 @@
...
+<!-- FAQ (optional) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT) licensed.
+
```
Comment (joel-kalema) [inline README.md:207]: - [ ] [OPTIONAL] Please, make sure you fixed the link to your license to make your README more professional 👍 


Screenshot 2023-11-01 144117

**Review by joel-kalema — APPROVED** [review]
Hi @gilberthappi 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# Congratulations! 🎉 


## Highlights
tables are well created 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**